### DISCUSSION TITLE
Example feed not returning any stops

### DISCUSSION THREAD

**Discussion by patricksheehan** [transitland/transitland#387] [Q&A] [answered]
https://transit.land/api/v2/rest/stops?feed_onestop_id=f-9q9-bart

^ this example which is in the documentation does not return any stops. Is this behavior expected? I'm trying to get a grasp for whether this API is legitimate and it's giving me pause...

**Comment by drewda** [comment, marked as answer]
We have recently switched coverage for SF Bay Area transit from individual agency feeds like that BART feed to the MTC Regional GTFS feed. We keep many individual agency feeds fetching for those who prefer to download GTFS feeds, but only import the Regional Feed for entity-level queries using the API.

Try your query again using the Onestop ID of the feed listed at https://www.transit.land/feeds/f-sf~bay~area~rg Or browse for any other feed with an imported and active feed version.

**Comment by patricksheehan** [comment]
Ah, ok. How could I self-service this information in the future? IE knowing whether a feed is “legit” and working as I would expect in the V2 API?Sent from my iPhoneOn Dec 19, 2022, at 7:06 PM, Drew Dara-Abrams ***@***.***> wrote:﻿
We have recently switched coverage for SF Bay Area transit from individual agency feeds like that BART feed to the MTC Regional GTFS feed. We keep many individual agency feeds fetching for those who prefer to download GTFS feeds, but only import the Regional Feed for entity-level queries using the API.
Try your query again using the Onestop ID of the feed listed at https://www.transit.land/feeds/f-sf~bay~area~rg Or browse for any other feed with an imported and active feed version.
